src/ios/FCMPlugin.m

```diff
@@ -106,14 +111,41 @@ -(void) notifyOfTokenRefresh:(NSString *)token
 {
     NSString * notifyJS = [NSString stringWithFormat:@"%@('%@');", tokenRefreshCallback, token];
     NSLog(@"stringByEvaluatingJavaScriptFromString %@", notifyJS);
-    
+
     if ([self.webView respondsToSelector:@selector(stringByEvaluatingJavaScriptFromString:)]) {
         [(UIWebView *)self.webView stringByEvaluatingJavaScriptFromString:notifyJS];
     } else {
         [self.webViewEngine evaluateJavaScript:notifyJS completionHandler:nil];
     }
 }
 
+-(void) upstream:(CDVInvokedUrlCommand *)command
+{
+    NSString* receiver = [NSString stringWithFormat:@"%@%@", FCM_PROJECT_SENDER_ID, FCM_SERVER_CONNECTION];
```

Comment:
Add a check if Sender ID is set or not. If not set throw an error with the appropriate message